vantage6/node/docker/docker_manager.py

```diff
@@ -126,8 +127,10 @@ def _set_database(self, config: Dict) -> None:
                 # We'll copy the file to the folder `data` in our task_dir.
                 self.log.info(f'Copying {uri} to {self.__tasks_dir}')
                 shutil.copy(uri, self.__tasks_dir)
+                uri = os.path.join(self.__tasks_dir, os.path.basename(uri))
```

Comment:
Isn't `self.__task_dir` a `Path` object?

Reply:
Yes it is, I edited the code to use those properties because it makes the code look neater